metaflow/metaflow_config.py

```diff
@@ -279,6 +279,15 @@
     },
 )
 
+###
+# Notify on success/error configuration
+###
+
+# Incident.io
+INCIDENT_IO_API_KEY = from_conf("INCIDENT_IO_API_KEY", None)
+INCIDENT_IO_SUCCESS_SEVERITY_ID = from_conf("INCIDENT_IO_SUCCESS_SEVERITY_ID", None)
+INCIDENT_IO_ERROR_SEVERITY_ID = from_conf("INCIDENT_IO_ERROR_SEVERITY_ID", None)
```

Comment:
we dont provide any of these as config options today for slack - can we lift them to cli args?